src/proto/Cardano.proto

```diff
@@ -33,6 +33,14 @@ message TxOutput {
     
     // ADA amount
     uint64 amount = 2;
+
+    // optional token amounts
+    repeated TokenAmount token_amount = 3;
+}
+
+// Collection of tokens with amount
+message TokenBundle {
```

Comment:
Maybe just `Tokens`?

Reply:
The reference Cardano implementations I checked use the `tokenBundle` term, so let's keep it that way for easier cross reference (e.g. CBOR encoding).